client/platform/web-girder/api/viame.service.ts

```diff
@@ -90,8 +90,8 @@ function runTraining(folderIds: string[], pipelineName: string, config: string)
 
 function saveMetadata(folderId: string, metadata: object) {
```

Comment:
Can the signature here be changed to `metadata: DatasetMetaMutable`?

I'd be nice to have typescript also enforce this.  Of course, it won't complain about superset (extra parameters) but it's better than `object`